src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/Constants.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerFx.LanguageServerProtocol
+{
+    internal class Constants
+    {
+        public class CodeActionKind
```

Comment:
no need for this inner classing. If you want this to be referenced as `LanguageServerProtocol.Constants.CodeActionKind.QuickFix` make a Constants folder and namespace

Reply:
yes simplified.